.gitattributes

```diff
@@ -0,0 +1,2 @@
+* text=auto
+*.less text eol=lf
```

Comment:
Probably not but I'd like to keep consistency against end-of-line and I found this practice on the [documentation](https://git-scm.com/docs/gitattributes#__code_eol_code) and in some popular repository too. Let me know if you want to keep or remove this addition. Thx.